builders/containerBuilder.go

```diff
@@ -0,0 +1,141 @@
+package builders
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/tfexplorer/models/generated/workloads"
+)
+
+// ContainerBuilder is a struct that can build containers
+type ContainerBuilder struct {
+	workloads.Container
+}
+
+// NewContainerBuilder creates a new container builder
+func NewContainerBuilder(nodeID string) *ContainerBuilder {
+	return &ContainerBuilder{
+		Container: workloads.Container{
+			NodeId: nodeID,
+		},
```

Comment:
For example a memory of 512 and cpu of 1 can be set by default. then customized later via the With* methods